README.md

```diff
@@ -484,6 +487,13 @@ Great way to see what might be going on when your server is lagging are timings.
 
 To get timings of your server you just need to execute the `/timings paste` command and click the link you're provided with. You can share this link with other people to let them help you. It's also easy to misread if you don't know what you're doing. There is a detailed [video tutorial by Aikar](https://www.youtube.com/watch?v=T4J0A9l7bfQ) on how to read them.
 
+However, when timings are on a % of the server thread is used by the timings itself. This can vary from 3-20%. Meaning if timings are not needed it is best to disable them either by `/timings off` (This would need to be done after every restart), or disabling timings in paper.yml
```

Comment:
I don't agree with this part at all, timings are not that heavy that you should turn them off by default.

Reply:
In my experience timings only ends up being heavy for large servers (80+ players). A server of that size should be able to read a spark report and know to disable them.